package.json

```diff
@@ -88,54 +85,59 @@
     "postcss": "^6.0.21",
     "prop-types": "^15.5.8",
     "react-intl": "^2.4.0",
-    "terra-base": "^5.0.0",
+    "react-router-dom": "^5.0.0",
     "terra-action-header": "^2.2.0",
-    "terra-application-layout": "^4.0.0",
+    "terra-aggregate-translations": "^1.0.0",
+    "terra-application-layout": "^5.1.0",
+    "terra-base": "^5.0.0",
     "terra-button": "^3.2.0",
     "terra-content-container": "^3.0.0",
     "terra-doc-template": "^2.2.0",
     "terra-icon": "^3.0.0",
     "terra-image": "^3.0.0",
     "terra-markdown": "^2.0.0",
-    "terra-navigation-layout": "^4.0.0",
     "terra-navigation-side-menu": "^2.0.0",
     "terra-search-field": "^3.2.0",
     "terra-slide-panel": "^3.0.0",
     "terra-theme-provider": "^3.0.0",
-    "terra-toolkit": "^4.5.0",
     "to-ast": "^1.0.0",
     "webpack-merge": "^4.1.2",
     "xfc": "^1.2.1"
   },
   "peerDependencies": {
-    "react": "^16.8.5",
     "react-dom": "^16.8.5",
-    "react-router-dom": "^4.2.2"
+    "react": "^16.8.5",
+    "terra-toolkit": "git://github.com/cerner/terra-toolkit.git#master",
+    "webpack": "^4.28.1"
```

Comment:
toolkit defined webpack peer dependencies. Do we want to here as well? Makes sense to

Reply:
I was on the fence about it, but i reasoned that since we define a webpack config here and add some webpack plugins, it makes sense to call out what version of webpack we support though terra-dev-site.